ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -171,12 +171,10 @@ class Endpoints(
   )(implicit lc: LoggingContextOf[InstanceUUID with RequestID]): A =
     withEnrichedLoggingContext(
       LoggingContextOf.label[JwtPayloadTag],
-      Map(
-        "ledger_id" -> jwtPayload.ledgerId.toString,
-        "act_as" -> jwtPayload.actAs.toString,
-        "application_id" -> jwtPayload.applicationId.toString,
-        "read_as" -> jwtPayload.readAs.toString,
-      ),
+      "ledger_id" -> jwtPayload.ledgerId.toString,
+      "act_as" -> jwtPayload.actAs.toString,
+      "application_id" -> jwtPayload.applicationId.toString,
+      "read_as" -> jwtPayload.readAs.toString,
```

Comment:
Great to see someone noticed this, I completely forgot that I should refactor that after I added the overload with the varargs instead of the need of using a map :smile:

Reply:
(This was a THANKS! :smile: )